src/status_im/navigation/core.cljs

```diff
@@ -91,27 +91,3 @@
  "bottom-sheet"
  (fn [] (gesture/gesture-handler-root-hoc views/bottom-sheet))
  (fn [] views/bottom-sheet))
-
-;; LEGACY (should be removed in status 2.0)
-
-(defonce
-  _
-  [(navigation/register-component
-    "popover"
-    (fn [] (gesture/gesture-handler-root-hoc views/popover-comp))
-    (fn [] views/popover-comp))
-
-   (navigation/register-component
-    "visibility-status-popover"
-    (fn [] (gesture/gesture-handler-root-hoc views/visibility-status-popover-comp))
-    (fn [] views/visibility-status-popover-comp))
-
-   (navigation/register-component
-    "bottom-sheet-old"
```

Comment:
i believe we still have the old sheet?

Reply:
Thank you @flexsurfer for review. Reverted code for `bottom-sheet-old` to be on safer side. (browser, etc. still uses this)